test/e2e/node/security_context.go

```diff
@@ -146,6 +179,88 @@ var _ = SIGDescribe("Security Context", func() {
 		testPodSELinuxLabeling(f, false, true)
 	})
 
+	addSeccompProfilesToNode := func(profiles []*seccompProfile) {
+		name := "seccomp-setup-" + string(uuid.NewUUID())
+		hostPathType := v1.HostPathDirectoryOrCreate
+		seccompProfileRoot := filepath.Join(services.KubeletRootDirectory, "seccomp")
```

Comment:
this should probably be `path.Join`, as we want a linux node path but the e2e.test binary could be run from windows ...

Reply:
Yep, changes to `path.Join`.